src/SQLite3-Core/SQLite3Library.class.st

```diff
@@ -868,7 +868,9 @@ SQLite3Library >> unix64ModuleName [
 SQLite3Library >> utf8StringToPharo: anUTF8String [
 	"Converts from SQLite UTF-8 to Pharo Multibyte Characters"
 	
-	^(ZnCharacterReadStream on: anUTF8String asByteArray readStream) upToEnd
+	^ SystemVersion current major >= 9
+		ifTrue: [ anUTF8String ]
+		ifFalse: [ (ZnCharacterReadStream on: anUTF8String asByteArray readStream) upToEnd ]
```

Comment:
I think a cleaner solution is to move this method as an extension in other packages (one for Pharo-9 and the other for Pharo<9) and update the baseline to load the correct version of the package depending on the Pharo version.

Reply:
Good suggestion, shall do...